src/routes/v2/authenticatedSites/settings.js

```diff
@@ -129,10 +130,12 @@ class SettingsRouter {
     router.post("/", attachRollbackRouteHandlerWrapper(this.updateSettingsPage))
     router.get(
       "/repo-password",
```

Comment:
Extremely basic qn: where exactly in the FE are we calling this ah? I couldnt find any corresponding one, the only enpoint that the corresponding [FE pr](https://github.com/isomerpages/isomercms-frontend/pull/1344) seems to call is `/sites/<siteName>/settings`, is this intended?

Reply:
The corresponding FE PR is only to restrict the privatisation route on the FE to email login as well! the call site is located in the base PR, https://github.com/isomerpages/isomercms-frontend/pull/1316